typedoc.cjs

```diff
@@ -0,0 +1,16 @@
+const packageJson = require("./package.json");
+// Pop last value out: packages/tests
+packageJson.workspaces.pop();
+
+module.exports = {
+  entryPointStrategy: "packages",
+  entryPoints: packageJson.workspaces,
+  out: "docs",
+  exclude: ["**/*.spec.ts"],
+  excludeInternal: true,
+  validation: {
+    invalidLink: true,
+    notExported: true,
+  },
+};
+packageJson.workspaces;
```

Comment:
Any specific reasons why we shifted from using `json` config to `js` config?

Reply:
So I could load the `entryPoints` from the `package.json`